topics/transcriptomics/tutorials/scrna-plant/tutorial.md

```diff
@@ -0,0 +1,534 @@
+
+
+## Data upload
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the two datasets from [Zenodo]({{ page.zenodo_link }}) or from the shared data library
+>
+>    ```
+>    {{ page.zenodo_link }}/files/GSE123818_Root_single_cell_shr_datamatrix.fixednames.transposed.csv.gz
+>    {{ page.zenodo_link }}/files/GSE123818_Root_single_cell_wt_datamatrix.fixednames.transposed.csv.gz
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Apply the tags `#shr` and `#wt` to one of the datasets, with respect to the initial labels. These tags will be inherited for each task that the dataset undergoes.
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> We can peek at the `#shr` file, and determine the dimensionality and naming scheme of the data. The rows and the columns depict different variables.
+>
+> 1. Which are the cell barcodes? Rows or Columns?
+> 1. Which are the gene names? Rows or Columns?
+> 1. How many cells in the dataset?
+> 1. How many genes in the dataset?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Rows, notice the long A-C-T-G based names.
+> > 1. Columns, "ATXG" are the gene prefixes commonly used by the Arabidopsis genomes.
+> > 1. There are 1 099 lines and therefore 1 099 cells
+> > 1. If we scroll the preview window all the way to the right we see 27 630 as the last number. Given that the first column are the cell names, we must subtract by 1 to get 27 629 cells.
+> >
+> {: .solution}
+>
+{: .question}
+
+If the above feels like a convoluted way to get the dimensionality, that's because we haven't imported the data into the right format. For this we need to import both datasets into a single AnnData object (see the [AnnData section in the previous tutorial]({% link topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md %}#anndata)).
+
+## CSV to AnnData
+
+> ### {% icon hands_on %} Hands-on: Converting the Data
+>
+> 1. {% tool [Import Anndata and loom](toolshed.g2.bx.psu.edu/repos/iuc/anndata_import/anndata_import/0.7.5+galaxy0) %} with the following parameters:
+>    - *"hd5 format to be created"*: `Anndata file`
+>        - *"Format for the annotated data matrix"*: `Tabular, CSV, TSV`
+>            - {% icon param-file %} *"Annotated data matrix"*: Multi-select both `SHR` and `WT` datasets
+>
+>    > ### {% icon tip %} Tip: Automatic
+>    >
+>    > We can now inspect the dimensionality of the dataset by "peeking" at the dataset in the history and observing the general information.
+>    {: .tip}
+>    
+>    > ### {% icon comment %} Comment
```

Comment:
maybe merge into the above tip box? Feels odd that there's two covering the same content

Reply:
oh whoops I didn't even notice the repeat here